Class 3 Homework

rawinslow24 opened the pull request:
homework1.html works as described. However, there's an issue where if the calculated time is either 12am or 12pm, this will not show as AM or PM and will not convert it to the appropriate time. I was unable to find a workaround.

homework2.html might be simplified, but for right now, it works.

homework3.html seems pretty simple enough.

rawinslow24 committed:
Add homework 1, 2, & 3 as separate .html files

Kilavagora reviewed (changes requested):
Use **const** unless your are re-declaring a variable, in that case use **let**. 

Your time calculation fails if time is within 12:00 AM - 1:00 AM. So better to keep it simple and just use 24 hour clock :P

Operators need to be surrounded by spaces. There needs to be a space after comma in the array. Easiest would be to use auto-formatter to avoid these issues :)

You have too much if elses in astro sign. Can you think of a way to make it simpler with a single if else?

rawinslow24 committed:
Update to homework1.html & homework2.html

On homework1.html, I converted it to 24hr format.

For homewrok2.html, I created a for loop and placed if statement inside of for loop and adjusted arrays to simplify the code.

Also spaced out commas in array, used let for variables that I'm re-assigning. I'll look into adding an auto formatter on Atom.
